samples/highcharts/website/small-demos-dashboards/demo.js

```diff
@@ -771,13 +771,9 @@ function datacursor() {
         };
     }
 
-    // Build table with Highcharts.Series aliases
+    // Build table with vegetables data
     function buildVegeTable() {
         const table = new DataTable({
-            aliases: {
-                name: 'vegetable',
-                y: 'amount'
-            },
             columns: {
```

Comment:
The same issue here. In general you can remove aliases, but the demos then need column assignments for each component.

Reply:
I will look into it.

Reply:
The demo doesn't work and is probably not maintained (judging by the number of commented out lines). I will enquire about the state of it before I make any effort to fix this issue.

Reply:
I fixed the demo in any case, several minor changes was enough.